lightly/utils/version_compare.py

```diff
@@ -0,0 +1,29 @@
+""" Utility method for comparing versions of libraries """
+
+# Copyright (c) 2020. Lightly AG and its affiliates.
+# All Rights Reserved
+
+
+def version_compare(v0: str, v1: str):
+    """Returns 1 if version of v0 is larger than v1 and -1 otherwise
+    
+    Use this method to compare Python package versions and see which one is
+    newer.
+
+    Examples:
+
+        >>> # compare two versions
+        >>> version_compare('1.2.0', '1.1.2')
+        >>> 1
+    """
+    v0 = [int(n) for n in v0.split('.')][::-1]
+    v1 = [int(n) for n in v1.split('.')][::-1]
+    assert len(v0) == 3
```

Comment:
I'd replace this by a `ValueError` (asserts for external checks is bad practice) and then catch the value error in the `fix_hydra_arguments` 🙂